File: trail1/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int val1 = 0;
        int val2 = 0;
        if (a <= b && a <= c) {val1 = 1;}
        if (a == b && a == c) {val2 = 1;}
        System.out.println(val1 + " " + val2);
    }
}

File: trail1/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] strlst = Console.ReadLine().Split();
        int a = int.Parse(strlst[0]);
        int b = int.Parse(strlst[1]);
        int c = int.Parse(strlst[2]);
        int val1 = 0;
        int val2 = 0;
        if (a <= b && a <= c) {val1 = 1;}
        if (a == b && a == c) {val2 = 1;}
        Console.WriteLine(val1 + " " + val2);
    }
}
